src/SOS/Strike/strike.cpp

```diff
@@ -6888,25 +6888,34 @@ DECLARE_API(Threads)
     // We need to support minidumps for this command.
     BOOL bMiniDump = IsMiniDumpFile();
 
-    if (bMiniDump && bPrintSpecialThreads)
-    {
-        Print("Special thread information is not available in mini dumps.\n");
-    }
-
     EnableDMLHolder dmlHolder(dml);
 
     try
     {
         Status = PrintThreadsFromThreadStore(bMiniDump, bPrintLiveThreadsOnly);
-        if (!bMiniDump && bPrintSpecialThreads)
+        if (bPrintSpecialThreads)
         {
-#ifdef FEATURE_PAL
-            Print("\n-special not supported.\n");
-#else //FEATURE_PAL    
-            HRESULT Status2 = PrintSpecialThreads(); 
-            if (!SUCCEEDED(Status2))
-                Status = Status2;
-#endif //FEATURE_PAL            
+            BOOL bSupported = true;
+
+            if (bMiniDump && bPrintSpecialThreads)
+            {
+                Print("Special thread information is not available in mini dumps.\n");
+                bSupported = false;
+            }
+
+            if (((g_pRuntime->GetRuntimeConfiguration() != IRuntime::WindowsCore) &&
+                (g_pRuntime->GetRuntimeConfiguration() != IRuntime::WindowsDesktop)))
+            {
+                Print("Special thread information is only supported on Windows dumps.\n");
+                bSupported = false;
+            }
+
+            if (bSupported)
+            {
+                HRESULT Status2 = PrintSpecialThreads();
```

Comment:
It looks like you need to ifndef FEATURE_PAL because PrintSpecialThreads only exists on Windows builds